roles/packages_installation/tasks/main.yml

```
@@ -5,10 +5,17 @@
       auto_import_keys: true
     when: ansible_facts['distribution'] == 'openSUSE Leap'
     loop: "{{ repos.opensuse }}"
+  - name: Install required packages on {{ ansible_facts['distribution'] }}
+    community.general.zypper:
+      name: "{{ packages.opensuse }}"
+      state: present
+      force_resolution: true
+    when: ansible_facts['distribution'] == 'openSUSE Leap'
   - name: Install required packages on {{ ansible_facts['distribution'] }}
     package:
-      name: "{{ packages.ubuntu if ansible_facts['distribution'] == 'Ubuntu' else packages.opensuse }}"
+      name: "{{ packages.ubuntu }}"
       state: present
+    when: ansible_facts['distribution'] != 'openSUSE Leap'
```

```suggestion
    when: ansible_facts['distribution'] == 'Ubuntu'
```
To avoid possible errors with different distros